packages/sprintf/index.js

```diff
@@ -94,6 +94,6 @@ export default function sprintf( string, args ) {
 
 		// To avoid "undefined" concatenation, return empty string if no
 		// placeholder substitution can be performed.
-		return value || '';
+		return value !== undefined && value !== null ? value : '';
```

Comment:
It makes me wonder what specific criteria we're using to determine whether a falsey value should be interpreted as an empty string vs. used verbatim. If `null` is coerced to an empty string, then why isn't `false`?

Looking at Jed, I finally understand now why the difference in behavior between Jed's `sprintf` and the standard `sprintf-js` library:

https://github.com/messageformat/Jed/blob/351c47d5c57c5c81e418414c53ca84075c518edb/jed.js#L406-L410

...which matches the behavior you've proposed here (perhaps intentionally). At least in this regard we've struck high compatibility between this and Jed.

I wonder if it's something we ought to at least document, since it doesn't seem self-evident and perhaps prone to some confusion.

Reply:
> It makes me wonder what specific criteria we're using to determine whether a falsey value should be interpreted as an empty string vs. used verbatim. If `null` is coerced to an empty string, then why isn't `false`?

As you point out below, this is mostly a choice for the sake of compatibility with `Jed` (to reduce migration risks), although it does make sense to me that nullish values (i.e., `null` and `undefined`) be treated differently from more semantic falsy values.

> I wonder if it's something we ought to at least document, since it doesn't seem self-evident and perhaps prone to some confusion.

That sounds like an excellent idea. Will update the readme.

Reply:
Readme updated.